opsdroid/tests/test_connector.py

```diff
@@ -0,0 +1,145 @@
+"""Test the opsdroid connector base class."""
+import pytest
+import asyncio
+
+import asynctest.mock as amock
+
+from opsdroid.core import OpsDroid
+from opsdroid.connector import Connector, register_event
+from opsdroid.events import Message, Reaction
+from opsdroid.cli.start import configure_lang
+
+configure_lang({})
+
+
+@pytest.fixture(scope="session")
+def loop():
```

Comment:
This fixture should be event_loop
https://github.com/pytest-dev/pytest-asyncio#event_loop
Consider moving this fixture to `conftest.py`

Reply:
yeah, sure. then we don't even need this fixture. its already implemented for us

Reply:
According to pytest-asyncio README, we should create a new event loop fixture to avoid using the default one.
https://github.com/pytest-dev/pytest-asyncio/blob/86cd9a6fd2/pytest_asyncio/plugin.py#L169-L174
So we probably need it, but only a definition in pytest

Reply:
okay will do.

Reply:
I am not sure, if this is correct. What I did is only copy the same code as they have, conftest now has this:
```
@pytest.yield_fixture
def event_loop():
    """Create an instance of the default event loop for each test case."""
    loop = asyncio.get_event_loop_policy().new_event_loop()
    yield loop
    loop.close()
```
Do I have to create a new custom event loop? If so, do you have any wishes how should it look like?